### PR TITLE
Update Test.cs

### PR DESCRIPTION
**Pull request by haithamabass**
add attributes

### CODE REVIEW

**Comment by haithamabass** [comment]
take care

File: ManagementSystem/Models/Test.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 {
     public class Test
     {
+        int buo { get; set; }
```
Comment (haitham-30) [inline ManagementSystem/Models/Test.cs:5]: oooh

**Review by haitham-30 — COMMENTED** [review]
ooohh yeah

File: ManagementSystem/Models/Test.cs (hunk 1 above)
Comment (haithamabass) [reply ManagementSystem/Models/Test.cs:5]: Will do

**Pull request merged**